### PR TITLE
New options to convert command

### LINKED ISSUE #228: Better convert options
Some suggestions for the CLI convert command:

* Possibility to choose fields to *include* instead of *exclude*
* Output to any file you want or stdout, in just one of the formats
* Make countries.php executable and register it as a binary in composer.json so we get a symlink in ./vendor/bin/

We pre-generate smaller JSON files with only the attributes we need, so they are faster to parse. We know what fields we do want so excluding everything else is an uncomfortable workaround.

### PR DESCRIPTION
**Pull request by emilv**
Fix #228

### CODE REVIEW

**Comment by emilv** [comment]
*\poke\*

**Comment by mledoze** [comment]
Hello @emilv, thank you for this pull request. I'm sorry for the delay, I have been busy lately, I will look into it this weekend.

Thank you for your patience.

File: README.md [hunk 1]
```diff
@@ -137,18 +137,45 @@ composer install
 After this finishes, run the following command (here we will exclude the `tld` field from the output, but you can exclude any field you want):
 
 ```sh
-php countries.php convert --exclude-field=tld
+composer exec countries.php convert --exclude-field=tld
```
Comment (mledoze) [inline README.md]: Why did you change `php` to `composer exec`?

File: src/MLD/Console/Command/ExportCommand.php [hunk 2]
```diff
@@ -61,8 +61,29 @@ protected function configure() {
...
+				'i',
+				InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
+				'If set, include only these top-level fields with the given name from the output. Cannot be used with --exclude-field',
+				[]
+			)
+			->addOption(
+				'format',
+				'f',
+				InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
+				'Formats to output.',
```
Comment (mledoze) [inline src/MLD/Console/Command/ExportCommand.php]: Could you please change this to `Output format`

**Comment by mledoze** [comment]
@emilv thank you for this good work! I commented on this pull request, could you look into it? 
Thanks

**Comment by mledoze** [comment]
@emilv friendly ping :) Did you see my comments on your pull request?

File: README.md (hunk 1 above)
Comment (emilv) [reply README.md]: Brainfart, mostly. I thought it would be nice to call the binary through Composer, but I regret it because it makes everything seem more complicated. Will revert.

File: src/MLD/Console/Command/ExportCommand.php (hunk 2 above)
Comment (emilv) [reply src/MLD/Console/Command/ExportCommand.php]: Sure.

**Comment by emilv** [comment]
Done!

**Comment by mledoze** [comment]
It's merged, thank you @emilv! 😄

**Comment by mledoze** [comment]
I will release a new version soon.
